frontend/src/types.ts

```diff
@@ -0,0 +1,8 @@
+export type User = {
+  _id: string;
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/23#issuecomment-2381248391>#09d02c</a><span> - 09/29/2024, 07:58 am</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The _id property is typed as string, which may not accurately represent MongoDB's ObjectId type.
</div>
<div id="fix">
<b>Fix:</b> Change the type of _id to more accurately represent MongoDB's ObjectId type.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,6 +1,8 @@
 +import { ObjectId } from 'mongodb';
 +
  export type User = {
 -  _id: string;
 +  _id: ObjectId;
    name: string;
    email: string;
    addressLine1: string;
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged